ci/android.groovy

```diff
@@ -36,7 +36,7 @@ def bundle() {
     }
   }
   sh 'find android/app/build/outputs/apk'
-  def outApk = "android/app/build/outputs/apk/${target}/app-${target}.apk"
+  def outApk = "android/app/build/outputs/apk/release/app-release.apk"
```

Comment:
same question here: is it just for this PR or we will always build release builds now?

Reply:
yes, that's just for PR. If we want to compare perf changes on startup we need to compare release build vs release build, otherwise it doesn't make too much sense